src/java.desktop/unix/classes/sun/print/PrintServiceLookupProvider.java

```diff
@@ -126,15 +127,15 @@
             }
         }
 
-        osname = java.security.AccessController.doPrivileged(
-            new sun.security.action.GetPropertyAction("os.name"));
+        osname = System.getProperty("os.name");
```

Comment:
I've seen it elsewhere. For example:
https://github.com/openjdk/jdk/blob/3789983e89c9de252ef546a1b98a732a7d066650/src/java.desktop/unix/classes/sun/awt/X11/XScrollbarPeer.java#L78 and https://github.com/openjdk/jdk/blob/6765f902505fbdd02f25b599f942437cd805cad1/src/java.desktop/share/classes/sun/awt/FontConfiguration.java#L135.

That said, I'll revert it because it's a behavior change.

Reply:
New commit pushed.